server/models/task.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Task extends Model {
+    static associate(models) {
+      this.belongsTo(models.User);
+    }
+  };
+  Task.init({
+    title: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        notNull: {
```

Comment:
kalau args:true kan udah default 
jadi bisa gausah ditulis
```suggestion
        notNull: {
              msg: "Title is required."
        },
```